src/components/SearchBar.js

```diff
@@ -0,0 +1,35 @@
+import { useState} from "react";
+
+
+const SearchBar = ({ onSubmit }) => {
+  
+  const [userInput, setUserInput] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    onSubmit(userInput); // prevent default and call the fetchevents function with userinput as an argument
+
+    setTimeout(() => {
+      setUserInput("");
+    }, 10000); // to make the input text dissapear 
+  };
+  const handleChange = (e) => {
+    setUserInput(e.target.value);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="searchGenreArtist">search</label>
+      <input
```

Comment:
you can try to put a placeholder in input so that you wont have to have a label show up on the page.

```js
<input placeholder="Search for an artist or a genre" />
```

and then you can just remove the search from label.